modules/osd-persistent-storage-csi-efs-sts.adoc

```diff
@@ -6,11 +6,11 @@
 
 :_content-type: PROCEDURE
 [id="efs-sts_{context}"]
-= Configuring AWS EFS CSI Driver Operator with Secure Token Service
+= Configuring Amazon EFS CSI Driver Operator with Secure Token Service
 
-This procedure explains how to configure the AWS EFS CSI Driver Operator with {product-title} on AWS Secure Token Service (STS).
+This procedure explains how to configure the Amazon EFS CSI Driver Operator with {product-title} on AWS Secure Token Service (STS).
 
-Perform this procedure before you have installed the AWS EFS CSI Operator, but not yet installed the AWS EFS CSI driver as part of the _Installing the AWS EFS CSI Driver Operator_ procedure. 
+Perform this procedure before you have installed the Amazon EFS CSI Operator, but not yet installed the Amazon EFS CSI driver as part of the _Installing the Amazon EFS CSI Driver Operator_ procedure. 
```

Comment:
Revoked.

Reply:
Updated, "Perform this procedure before you have installed the AWS EFS CSI Operator, but not yet installed the link:https://github.com/openshift/aws-efs-csi-driver[AWS EFS CSI driver] as part of the _Installing the AWS EFS CSI Driver Operator_ procedure. "